generated/nifake/nifake/session.py

```diff
@@ -90,15 +90,20 @@ def __exit__(self, exc_type, exc_value, traceback):
 
```

Comment:
I think we need to add something in metadata of properties/methods that can be called with nested repeated capabilities, so that the docstring for them shows users that they can nest repeated capabilities and how to do it.

Reply:
I would rather not do that. Instead I added to the static nidigital documentation some information about nesting repeated capabilities. I am open to how to better explain what we are doing though.